### PR TITLE
Issue 440 create individual user space

### PR DESCRIPTION
**Pull request by bhavika-tibrewal**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce.

# Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

# Screenshots or example output

image

### CODE REVIEW

File: src/common/contributor_profile/EditProfile.jsx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  FormControl,
+  MenuItem,
+  Select,
+  TextField,
+} from "@mui/material";
+import { useNavigate } from "react-router-dom";
+import { useSkillsList } from "common/hooks/useSkillsList";
+
+const expert_levels = [
+  {
```
Comment (atapas) [inline src/common/contributor_profile/EditProfile.jsx:14]: Why it should be hardcoded and not coming from a GraphQL call?

File: src/common/services/user_profile.js [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+          field: "id",
+          operator: "eq",
+          value: id,
+        },
+      ],
+    },
+  };
+  return payload;
+};
+export const InsertUserProfile = (id, avatarUrl) => {
```
Comment (atapas) [inline src/common/services/user_profile.js:19]: For all these operations, please provide comments.

File: src/common/services/user_skill_map.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+};
+
+const InsertUserSkillMap = (user_id, skill_id, expert_level) => {
+  return {
+    display: "insert User Skill Map",
+    name: "insert_meta_user_skill_map_one",
+    function: "insert_meta_user_skill_map_one",
+    write: true,
+    object: {
+      user_id: "03724f8f-e5c7-423d-a98c-a7fdee9e959f",
```
Comment (atapas) [inline src/common/services/user_skill_map.js:26]: Hoping this is only for the test purpose and the hard coding will be removed later?

File: src/common/header/HeaderNav.jsx [hunk 4]
```diff
@@ -18,6 +20,13 @@ const HeaderNav = ({ showBrowse }) => {
   const [anchorEl, setAnchorEl] = useState(null);
 
   const open = Boolean(anchorEl);
+  const navigate = useNavigate();
+  const userId = useUserId();
+  const isAuthenticated = useAuthenticated();
```
Comment (atapas) [inline src/common/header/HeaderNav.jsx:25]: isn;t there another async alternate for it? Which one to use?

File: src/common/services/user_skill_map.js (hunk 3 above)
Comment (bhtibrewal) [reply src/common/services/user_skill_map.js:26]: yes just for test purpose, will be updated ACC to the param we will receive in the url

**Comment by atapas** [comment]
@bhavika-tibrewal / @NagarjunShroff Please resolve conflicts and pull main to get linting fixes

**Comment by atapas** [comment]
The build fails:

https://vercel.com/reactplayio/react-play/BQkVodsLSVSDm4Gj74JTP695Lmob

**Comment by atapas** [comment]
@bhavika-tibrewal Please resolve conflicts

**Comment by NagarjunShroff** [comment]
> @bhavika-tibrewal Please resolve conflicts

@atapas done.

**Pull request closed without merging**